app/src/main/java/de/westnordost/streetcomplete/quests/wheelchair_access/AddWheelchairAccessPublicTransportForm.java

```diff
@@ -21,6 +20,27 @@
 
 	private void setTitle()
 	{
-		setTitle(R.string.quest_wheelchairAccess_public_transport_title);
+		String highway = null;
+		OsmElement element = getOsmElement();
+		if (highway == null) {highway = element != null && element.getTags() != null ? element.getTags().get("highway") : null;}
+		if (highway == null) {highway = element != null && element.getTags() != null ? element.getTags().get("railway") : null;}
+		if (highway == null) {highway = element != null && element.getTags() != null ? element.getTags().get("public_transport") : null;}
+
+		switch (highway) {
+			case "bus_stop":
+				setTitle(R.string.quest_wheelchairAccess_public_transport_bus_stop_title);
+				break;
+			case "platform":
+				setTitle(R.string.quest_wheelchairAccess_public_transport_platform_title);
+				break;
+			case "station":
+				setTitle(R.string.quest_wheelchairAccess_public_transport_station_title);
+				break;
+			case "subway_entrance":
+				setTitle(R.string.quest_wheelchairAccess_public_transport_entrance_title);
+				break;
+			default:
+				setTitle(R.string.quest_wheelchairAccess_public_transport_default_title);
+		}
```

Comment:
I've changed the way the title is set to include both name and type where it's available. e.g Is the bus stop "Central" wheelchair accessible? or "Is this bus stop wheelchair accessible?" if there's no name.